Adding BrowserLikePolicyForHTTPS to __all__

pappacena opened the pull request:
On twisted.web.client, `BrowserLikePolicyForHTTPS` is not listed on `__all__`, although the user is explicitly told to use it on `WebClientContextFactory` comments, for example. The ticket on trac is here: https://twistedmatrix.com/trac/ticket/9769 .

I know this is a super small kind of irrelevant PR, but I'm proposing it for the sake of consistency and to suppress some warning messages I'm having in Launchpad.net development. :-)

pappacena committed:
Adding BrowserLikePolicyForHTTPS to __all__

On twisted.web.client, BrowserLikePolicyForHTTPS is not listed on __all__, although the user is explicitly told to use it on WebClientContextFactory comments, for example.

glyph commented:
@pappacena no concerns about a small PR! if this had followed the contributor checklist I'd probably go ahead and land it right now :)

cjwatson commented:
@pappacena said to me elsewhere that he was having trouble logging into Trac.  I've filed https://twistedmatrix.com/trac/ticket/9769 to match this PR, but @pappacena will still need to go through the rest of https://twistedmatrix.com/trac/wiki/TwistedDevelopment#SubmittingaPatch.

pappacena committed:
Adding newsfragment file for BrowserLikePolicyForHTTPS on its module's __all__ variable

pappacena commented:
Thanks for the ticket and the link for the "Submiting a Patch" page, @cjwatson .

@glyph , I've updated the trac ticket to mark it to be reviewd, and added the newsfragment file. All the CI checks passed now.

Thanks for the support!

glyph commented:
Thanks @pappacena !  I'm sorry to hear about the trac login issues. We're about an hour away from a big upgrade for the machine that hosts it, so hopefully issues like that will be less common.

glyph committed:
Merge branch 'trunk' into patch-1

glyph reviewed (approved):
Looking good.
